### ISSUE TITLE
Polish starts of the Understanding documents

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#395]
_From @alastc on April 24, 2018 23:24_

From an onlist discussion, @mbgower suggested using his [Simple Statements of new WCAG 2.1 SCs](https://www.ibm.com/blogs/age-and-ability/2018/02/08/simplifying-new-wcag-2-1-guidelines/) as the start of the intent section.

I think it's best to do towards the end of the understanding doc editing process, to  prevent clashes.

Also, it should be done on a case-by-case basis, some of the docs may have very well thought-out introductions. 

Adding this issue for tracking.

_Copied from original issue: w3c/wcag21#882_

**Comment by alastc** [comment]
@mbgower do you have these to hand? I.e. is now a good time to re-address this (good idea)?

**Comment by fstrr** [comment]
This issue appears to be inactive, so I'm closing it. If it needs to be re-opened, please do so.
